lib/double_entry/reporting/aggregate.rb

```diff
@@ -25,7 +25,7 @@ def amount(force_recalculation = false)
         end
       end
 
-      def formatted_amount(value = amount())
+      def formatted_amount(value = amount)
```

Comment:
@joneslee85 I'll merge this. Are you able to run it through your market 2.2.2 build please? Perhaps it's something we can fix on that end.